monero-outreach-docs/translations/pt-br/PSA/files/mining/mine_support_network_pt-br.md

```diff
@@ -0,0 +1,25 @@
+# Minere Monero para apoiar a rede
+
+Os mineradores do Monero usam hardwares de computação para executar uma variante do algoritmo CryptoNight, competindo por uma recompensa de mineração que é distribuída cada vez que um novo bloco é adicionado à Blockchain. A mineração do Monero é conhecida por sua resistência aos ASICs (circuitos integrados para aplicações específicas). Isso significa que ela é mais acessível e amigável à mineração utilizando processadores comuns (CPU) e placas de vídeo (GPU), no entanto, ela não consegue impedir que indivíduos e empresas criem os poderosos ASICs e FPGAs. Em função disso, os desenvolvedores do Monero fazem a cada 6 meses uma atualização de rede de rotina, com o objetivo de evitar que a mineração através de ASICs e FPGAs ganhe controle da maior parte do hashrate da rede.
```

Comment:
I think the text will be more fluent if you change the order of this sentence: "Em função disso, os desenvolvedores de Monero fazem uma atualização de rede de rotina a cada 6 meses, [...]"